### ISSUE TITLE
Disabling Bidirectional Isolation in Fluent

### ISSUE THREAD

**Issue by fosskers** [kellpossible/cargo-i18n#45]
This is a feature request that stems from discoveries in the following issues:

- https://github.com/kellpossible/cargo-i18n/issues/44
- https://github.com/projectfluent/fluent-rs/issues/172#issuecomment-731619772

I see that a `FluentLanguageLoader` has an inner `LanguageConfig`, a struct private to that module:
```rust
struct LanguageConfig {
    current_language: LanguageIdentifier,
    language_bundles: Vec<LanguageBundle>,
}

struct LanguageBundle {
    language: LanguageIdentifier,
    bundle: FluentBundle<Arc<FluentResource>>,
    resources: Vec<Arc<FluentResource>>,
}
```
The `FluentBundle` is what, by default, wraps injected arguments in the supposed-to-be-invisible directionality characters described in #44 . If one has direct control over the bundle, this wrapping can be disabled via:
```rust
bundle.set_use_isolating(false);
```
Could a config option be offered, perhaps a field in `i18n.toml`, to set this to false?

**Comment by fosskers** [comment]
The setting could perhaps be set after this line: https://github.com/kellpossible/cargo-i18n/blob/master/i18n-embed/src/fluent.rs#L329

**Comment by fosskers** [comment]
Oh, a method directly on `FluentLanguageLoader` might do it. I will experiment with this.

**Comment by kellpossible** [comment]
Implemented now in `i18n-embed` version `0.9.4`
